release-notes/WCF-Web-Service-Reference-notes.md

```diff
@@ -1,16 +1,17 @@
-# WCF Web Service Reference Provider
+# WCF Web Service Reference Provider - Release Notes
 
-## Information
-
-* [Usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
-* [Report an issue](https://github.com/dotnet/wcf/issues/new)
+WCF Web Service Reference connected services provider is now part of Visual Studio 2017. Please refer to [usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
+for instructions or [report an issue or give us feedback](https://github.com/dotnet/wcf/issues/new).
 
 ## Release History
 
-### 1.0.0 - Visual Studio 2017 [v15.5 Preview 1]
+### 10/11/2017 - Visual Studio 2017 v15.5 Preview 1.0 [27009.1.d15rel]
 
-* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.0.0.md)
+* WCF Web Service Reference connected services provider is included directly inside VS 2017 starting this release.
```

Comment:
included as part of VS ...
// 'included directly inside' sounds a bit redundant to me.

Reply:
Updated.